pages/coinbase/homepage.py

```diff
@@ -0,0 +1,30 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support import expected_conditions as EC 
+from selenium.webdriver.support.ui import WebDriverWait as wait
+from selenium.webdriver.common.by import By
+import secure.cookies
+import config
+
+
+
+class Home_Page():
+
+    def __init__(self,driver):
+        self.driver = driver
+        driver.get(config.coinbase_homepage_url)
+
+    #Buttons
+
+    # coinbase has two-factor authentication which involves sending a text
+    # via cellphone; it is optional to forego this authentication, so 
+    # cookies are added below to do this. It is currently unclear which cookie controls 
+    # the authentication. So, I just signed in, checked the option to not ask for
+    # the text code again, then signed out and grabbed all the cookies. Below I loaded 
+    # all these cookies which include the cookie/s necessary to forego this second 
+    # authentication.
+
+    def home_page_sign_in_button_loc(self): return (By.XPATH,"//a[@title='Sign in']")
```

Comment:
This doesn't need to be a function. just `sign_in_button_loc = (By.XPATH, "//a[@title='Sign in']")`

Also, don't use redundant naming. This function is `Home_Page.home_page_sign_in_button_loc()`; don't name functions in such a way that you have to type something (like "home page") twice.

Reply:
There are two sign in buttons one on the home page and one on the sign in page. But I guess since they are in separate files I don't need to distinguish them.

Reply:
The class name distinguishes them already. to use the button on the home page, you have to use HomePage.sign_in_button. to use the button on the sign_in_page you have to use SigninPage.sign_in_button.